cmd/validate.go

```diff
@@ -24,23 +24,29 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var validateCmd = &cobra.Command{
-	Use:     "validate [flags] [args]",
-	Short:   "Check for validation and parse errors",
-	Long:    `Check for validation and parse errors.`,
-	Example: "  gauge validate specs/",
-	Run: func(cmd *cobra.Command, args []string) {
-		setGlobalFlags()
-		if err := isValidGaugeProject(args); err != nil {
-			logger.Fatalf(err.Error())
-		}
-		initPackageFlags()
-		track.Validation()
-		validation.Validate(args)
-	},
-	DisableAutoGenTag: true,
-}
+var (
+	validateCmd = &cobra.Command{
+		Use:     "validate [flags] [args]",
+		Short:   "Check for validation and parse errors",
+		Long:    `Check for validation and parse errors.`,
+		Example: "  gauge validate specs/",
+		Run: func(cmd *cobra.Command, args []string) {
+			setGlobalFlags()
+			validation.HideSuggestion = hideSuggestion
+			if err := isValidGaugeProject(args); err != nil {
+				logger.Fatalf(err.Error())
+			}
+			initPackageFlags()
+			track.Validation(hideSuggestion)
+			validation.Validate(args)
+		},
+		DisableAutoGenTag: true,
+	}
+	hideSuggestion bool
+)
 
 func init() {
 	GaugeCmd.AddCommand(validateCmd)
+	validateCmd.Flags().BoolVarP(&hideSuggestion, "hide-suggestion", "", false, "Gives default step implementation for un implemented steps")
```

Comment:
"unimplemented" is a single word. Also lets not call it "default" step implementation. How about message like this:

"Prints a step implementation stub for every unimplemented step"